docs/fixed_agg_workflow.md

```diff
@@ -0,0 +1,196 @@
+
+```
+from cladecombiner import pango_sc2_nomenclature as pn
+
+pn.setup_alias_map()
+tree = pn.taxonomy_tree(taxa)
+
+scheme = cladecombiner.PhylogeneticTaxonomyScheme(tree)
+```
+
+These trees are wrappers around `dendropy.Tree` objects, and can be printed to screen for inspection.
+The command `print(tree.as_ascii_plot(plot_metric="level", show_internal_node_labels=False))` will, for these taxa, yield
+
+```
+                         /----+---- BA.5.2.6
+                    /----+
+                    |    \----+----+----+----+----+----+----+---- BQ.1.1.23
+                    |
+                    |                   /----+----+----+---- KP.1.1
+                    |              /----+
+                    |              |    \----+---- JN.1.9.1
+                    |              |
++----+----+----+----+         /----+---- XDQ
+                    |         |    |
+                    |         |    |---- JN.9
+                    |    /----+    |
+                    |    |    |    \---- JN.6
+                    |    |    |
+                    |    |    \---- BA.2.86
+                    \----+
+                         |----+----+----+----+----+----+----+----+----+---- FW.1.1.1
+                         |
+                         |----+----+----+----+---- XCU
+                         |
+                         \---- BA.2
+```
+
+The observed taxa are all tips, and the internal nodes are represented by `+` (names not shown for compactness, set `show_internal_node_labels=True` to see these).
+Some of these nodes here have only one child, while others have many, because cladecombiner makes this tree to include every named Pango lineage in the complete histories of all lineages.
+Thus, the first four nodes are the root (in cladecombiner parlance, an empty string, `""`), B, B.1, B.1.1, and B.1.1.529 (aka BA, but note that cladecombiner is pedantic about names and does not accept naked aliases like BA).
+Of these, only BA has multiple children among the observed lineages, so only it is a branching point in the tree.
+
+## Aggregation targets
+
+We also need the desired aggregation targets, the taxa to which we want to aggregate.
+As with observed taxa, we can read these in or create them from strings.
+Let us assume we want to aggregate to BA.2.86.1 (aka JN), BA.2, BA.5, and KP.1.1.
+
+Aggregation requires us to be explicit about whether we are truly aggregating or not.
+Three of these represent actual aggregations of the observed taxa, as we have observed lineages which are children of BA.2, BA.5, and JN.
+However, as we have no children of KP.1.1 in our observed lineages, it is a tip.
+
+```
+target_taxa = [
+    cladecombiner.Taxon("BA.2", False),
+    cladecombiner.Taxon("BA.2.86.1", False),
+    cladecombiner.Taxon("BA.5", False),
+    cladecombiner.Taxon("KP.1.1", True),
```

Comment:
Is this equivalent to saying "map KP.1.1 to itself, and not to anything above it"?

Reply:
Yes. Adding that to the text above for clarity, and cross-linking to the section we describe _why_ we have to do this.